datafusion/common/src/utils/mod.rs

```diff
@@ -769,6 +769,21 @@ pub fn set_difference<T: Borrow<usize>, S: Borrow<usize>>(
         .collect()
 }
 
+/// Checks whether the given index sequence is monotonically non-decreasing.
+#[deprecated(since = "45.0.0", note = "Use std::iterator::is_sorted instead")]
```

Comment:
```suggestion
#[deprecated(since = "45.0.0", note = "Use std::Iterator::is_sorted instead")]
```

I double checked and this I think is capitalized `Iterator` -- so updating it: https://doc.rust-lang.org/nightly/std/iter/trait.Iterator.html#method.is_sorted